components/Trade/Advanced/TourSteps/index.tsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Prompt = styled.div`
+    background: var(--color-accent);
+`;
+const HelperTitle = styled.span`
+    display: flex;
+    align-items: center;
+    height: 57px;
+    padding: 0px 16px;
+    font-size: var(--font-size-large);
+    border-bottom: 1px solid var(--color-background-secondary);
+    pointer-events: none;
+`;
+const HelperText = styled.p`
+    padding: 16px 16px;
+    font-size: var(--font-size-small);
+    pointer-events: none;
+    margin-bottom: 6px;
+
+    span {
+        display: block;
+        margin-bottom: 7px;
+    }
+`;
+
+const HelperList = styled.ul`
+    list-style: inherit;
+    padding-inline-start: 16px;
+`;
+
+// const defaultStyles = {
+//     backgroundColor: "var(--color-accent)",
+//     color: "white",
+//     padding: '0px',
+//     overflow: 'hidden',
+// };
+
+export const tourConfig = [
+    // Wallet button
+    {
+        selector: 'div[class^="AccountDropdown__MainButton"]',
+        content: () => (
+            <Prompt>
+            <HelperTitle>Connect to Arbitrum</HelperTitle>
+            <HelperText>
+                Connect your wallet to Arbitrum Mainnet to get started with Tracer. 
+            </HelperText>
+            </Prompt>
+        ),
+        action: () => {
+            const calculatorEl = document.querySelector('div[class*="Calculator"]') as HTMLElement;
+            const marginModalEl = document.querySelector('div[class*="AccountModal"]') as HTMLElement;
+            if(calculatorEl){
+                calculatorEl.style.display = 'block';
+                calculatorEl.style.zIndex = '100000';
+            }
+            if(marginModalEl){
+                marginModalEl.style.display = 'block';
+            }
+        },
+        position: 'bottom',
+        style: {
+            top: '50px',
+            left: '0px',
+        }
+    },
+    // Calculator modal
+    {
+        selector: 'div[class*="Calculator"] .content',
+        content: () => (
+          <Prompt>
+            <HelperTitle>Calculate Margin Required</HelperTitle>
+            <HelperText>
+                Use the calculator to see how much exposure and leverage you can get with your margin, and the price you'd be liquidated at. Once you've finished playing around, click 'Deposit'. 
+            </HelperText>
+          </Prompt>
+        ),
+        action: () => {
+            const calcButton = document.querySelector('button[class*="Account__CalculatorButton"]') as HTMLElement;
```

Comment:
I think we have to be careful with these classes ie Account__CalculatorButton since Account__CalculatorButton is generated from style components. But if we change the name from Account to Portfolio for example it auto generates it to be Porfolio__CalculatorButton. Just something to keep in mind. The alternative would be to just use Id's on the components you want to select. That way it will only ever change if the id of that component changes

Reply:
I'll update the selector components with some data-attributes